### ISSUE TITLE
BAD RoboTwin  RESULTS with QWENPI

### ISSUE THREAD

**Issue by hanyangyu1021** [starVLA/starVLA#212]
I am currently evaluating Qwen-Pi on the Robotwin benchmark using 8× NVIDIA H20 GPUs. After training for 40k steps with a total batch size of 32 (8 GPUs × 4 BS), the model exhibits severe jittering/shaking behavior during inference.

Compared to Pi0, which typically achieves reasonable stability and performance under similar computational budgets, Qwen-Pi fails to produce smooth trajectories. I would like to clarify whether this is a convergence issue (insufficient training) or a potential misconfiguration in the action head/inference settings.

<img width="1418" height="1322" alt="Image" src="https://github.com/user-attachments/assets/f07dcfed-b504-4adc-b0dd-81429fe9d234" />

<img width="2190" height="1819" alt="Image" src="https://github.com/user-attachments/assets/60c29dd9-42a5-41ef-b4a9-f181b5000a44" />

https://github.com/user-attachments/assets/d771429d-652c-4eda-9aaf-b56f703dd9cd

**Comment by hanyangyu1021** [comment]
I have also tested the official Qwen-OFT ckpt, and the resulting trajectories are significantly smoother. I would appreciate any insights or guidance on why the standard fine-tuning exhibits such severe jitter. Thanks in advance!

**Comment by Yysrc** [comment]
I have also reproduced the same experiment. First, you should use the latest version of the code (with output layer normalization for actions), which results in smoother trajectories. Second, the default four-step denoising in Qwen-PI is insufficient; switching to ten or more steps will yield better performance.

**Comment by JinhuiYE** [comment]
Congrats!
